### ISSUE TITLE
Regexp doesn't match if special chars at beginning or end of word

### ISSUE THREAD

**Issue by kometenstaub** [nothingislost/obsidian-highlight-current-word#3]
I've noticed that it doesn't highlight the word if there is a special character (`é`, `à`, `ü`) at the beginning or end of a word.

![highlight-current-word](https://user-images.githubusercontent.com/83140328/147654291-0609b4e5-531f-4ec8-8819-b0a0c6f50143.gif)

I looked at the code and it does get the word correctly, but it doesn't apply the `span` with the CSS class to it.

```js
const view = app.workspace.activeLeaf.view
const word = view.editor.cm.state.wordAt(view.editor.cm.state.selection.main.from)
let wordText = word && view.editor.cm.state.doc.sliceString(word.from, word.to)
```
![image](https://user-images.githubusercontent.com/83140328/147654348-8caf3f8a-afab-411b-a9f3-d502463c5191.png)

The issue seems to be the `\b` in the regular expression:

![image](https://user-images.githubusercontent.com/83140328/147654858-b325de01-bd64-42c1-b6d8-fac9ad809777.png)
![image](https://user-images.githubusercontent.com/83140328/147654880-123023ca-e1ff-456d-8473-4d400e683009.png)
![image](https://user-images.githubusercontent.com/83140328/147654941-fa58b32a-d443-4657-876e-d209a543e62f.png)
![image](https://user-images.githubusercontent.com/83140328/147654953-05257bb0-95bf-43e6-99af-04f8d7deeaf2.png)

**Comment by chrisgrieser** [comment]
Not sure how to implement that correctly, but maybe the regexes used by the better word count plugin helps with this issue? 🤔 

https://github.com/lukeleppan/better-word-count/blob/master/src/data/stats.ts

**Comment by nothingislost** [comment]
Addressed in 0.0.5

**Comment by kometenstaub** [comment]
It works well now, thank you so much! :heart:
